packages/components/src/Link/Link.tsx

```diff
@@ -34,7 +36,7 @@ const Link = forwardRef<HTMLAnchorElement, LinkProps>(
     ref,
   ) => {
     return (
-      <Clickable
+      <ClickableStyleWrapper
```

Comment:
Note to self: the dosstring will need to be updated if https://github.com/chanzuckerberg/edu-design-system/pull/628 merges first.